doc/release/2021-08-releases.rst

```diff
@@ -49,28 +52,140 @@ Core
 
 
 -   The information about the state of synchronous replication is now available via the
-    ``box.info.synchro`` interface (:tarantool-release:`2.7.3`, :tarantool-issue:`5191`).
+    ``box.info.synchro`` interface
+    (:tarantool-release:`2.7.3`, :tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5191`).
+
+-   Field type UUID is now part of field type ``SCALAR``
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`6042`).
+
+-   Field type ``UUID`` is now available in SQL, and a new UUID can be generated
+    using the new SQL built-in function ``UUID()``
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5886`).
+
+-   **[Breaking change]** The ``timeout()`` method of the ``net.box`` connection,
+    marked deprecated more than four years ago (in ``1.7.4``), has been dropped.
+    It negatively affected the performance of hot ``net.box` methods, such as ``call()`` and ``select()``
+    if those were called without specifying a timeout
```

Comment:
```suggestion
    if they were called without a specified timeout
```
otherwise it looks like the methods specify the timeouts by themselves, it's not the user who does it